### ISSUE TITLE
Crossing loading zones is very long

### ISSUE THREAD

**Issue by ohc33** [TheSimpleBlue/gimimprove#2]
The time it takes to load a new area is drastically increased from nearly instant to 3 seconds. It makes using the plugin not worth it.

**Comment by TheSimpleBlue** [comment]
Thank you for taking the time to fill out feedback and using the plugin. I was powering through it since I figured I was the only one using it, but I'll try to address this in the next few days after work (I just found the issues tab today, which I figured would notify me via email but it didn't). Sorry to hear if you aren't using it anymore, but I'll update this once it's fixed anyway :)

**Comment by Joshmesh** [comment]
Yeah having this issue too same with teleporting. I just found this plugin too shame it has this issue sadly game is not playable with this hope you are able to sort it out.
